incl/header.html

```diff
@@ -3,7 +3,7 @@
 <html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
 
 <head>
-  <title><#TITLE> | The Coq Proof Assistant</title>
+  <title><#TITLE> | The Coq/Rocq Proof Assistant</title>
```

Comment:
Keep in mind that Inria has an official trademark on the name Coq (which is recognized in both USA and China, for what that is worth). I have only rudimentary notions on what that implies and it would have to be double-checked with Inria lawyers, but I am pretty sure that such a prominent modification is the fastest way to lose the trademark on Coq.

Reply:
My understanding (not a lawyer, etc.):
- Existing versions of Coq released under that name are covered by a Coq trademark and would remain so after a rename.
- Trademark rights may be lost if the trademark is not used anymore for a period of time (Wikipedia says [five years](https://en.wikipedia.org/wiki/Trademark#Maintaining_rights)), so (1) any change would only take effect after a few years, when presumably a rename would be deeply in progress and (2) if the website uses a dual naming scheme, Coq *and* Rocq, then we are still using the trademark and the rights may well be maintained.

Who has asked INRIA lawyers about the rename, and what did they say?

Reply:
Writing somewhere on the website that "the Coq proof assistant is also known as the Rocq proof assistant" is presumably fine. Writing "the Coq/Rocq proof assistant" in the title of every single page of the website is no longer a case of dual naming. It is pure single naming.

That said, I have always been a proponent of keeping the Coq prefix in the new name (e.g., Coqatrix) to ease the transition process, but the idea never got enough traction. So, having the tool be renamed Coq/Rocq (Coqroach?) would certainly have my favor. But this ship has long sailed and the plan agreed upon is to get Coq to be renamed into Rocq, not into Coq/Rocq.

As for disclosing on a public forum what was discussed with the lawyers, I would not dare. I suggest you directly phone our [coordinator](https://coq.inria.fr/coq-team.html) to get a feel of how far the lawyers have progressed on the topic.

Reply:
I don't think that people would confuse Coq/Rocq for a single name, but I am happy to try to find a wording that would please you, in that it is clear that Coq and Rocq are both acceptable names for the system.

I don't particularly care about trademarks for open source project myself -- I think that it's probably a complete waste of time and effort -- and I am happy to ignore anything related to them, instead of having to phone people to learn secret information about the project. It is perfectly fine if the Coq stakeholders decide that trademark protection is important to them and they do stuff along those lines, as long as it remains orthogonal to the healthy development of the project. Deciding a rename, involving the community in a major way to decide a better name, officially announcing that a choice has been made, *and then doing nothing externally visible at all for months* is not healthy behavior for an open-source project.

Reply:
I just pushed a new version. Instead of using "The Coq/Rocq Proof Assistant", the new version says "The Coq (Rocq) Proof Assistant". Initially I wrote "The Coq Proof Assistant, the Rocq Prover", but this was too large and the banner would overflow on larger zoom level.

The wording in the right menu also does not use Coq/Rocq anymore:

> During this transition period, please feel free to use both names, for example by writing "formalized with Rocq (formerly Coq)" or "formalized with Coq (also named Rocq)".

Reply:
@tabareau evaluates the estimal arrival time to December. Maybe better to stay on the safe side and be explicit on that, in the style: "planned to be renamed the Rocq prover by the end of 2024".

Reply:
Here is what I had before (in full):

> October 2023: we have decided to rename 'Coq' into 'The Rocq Prover'. Renaming all existing resources is a lot of work, it will be a slow, gradual transition. During this transition period, please feel free to use both names, for example by writing "formalized with Rocq (formerly Coq)" or "formalized with Coq (also named Rocq)".

Would you prefer the following wording, which adds a mention of the 2024 timeline?

> October 2023: we have decided to rename 'Coq' into 'The Rocq Prover'. We hope to perform the rename in 2024, but renaming all existing resources is a lot of work so it will be a slow, gradual transition.  During this transition period, please feel free to use both names, for example by writing "formalized with Rocq (formerly Coq)" or "formalized with Coq (also named Rocq)".

(Note: I would not recommend any formulation that make it sounds like a complete rename will occur in 2024. This deadline is probably going to be missed, and it's fine, and announcing it too loudly can add pressure to the process.)